fec.config.js

```diff
@@ -0,0 +1,28 @@
+const { resolve } = require('path');
+
+module.exports = {
+  appUrl: '/insights/tasks',
+  debug: true,
+  useProxy: process.env.PROXY === 'true',
+  proxyVerbose: true,
+  plugins: [],
+  ...(process.env.HOT
+    ? { hotReload: process.env.HOT === 'true' }
+    : { hotReload: true }),
+  ...(process.env.port ? { port: parseInt(process.env.port) } : {}),
+  moduleFederation: {
+    shared: [
+      {
+        'react-router-dom': {
+          singleton: true,
+          import: false,
+          version: '^6.8.1',
```

Comment:
let's link it to package.json as we did in Compliance https://github.com/RedHatInsights/compliance-frontend/blob/master/fec.config.js#L26C1-L27C1